addon/components/radio-group.hbs

```diff
@@ -0,0 +1,32 @@
+<input type='hidden' value={{this.selectedValue}} />
+{{! template-lint-disable no-positive-tabindex}}
```

Comment:
Are these still strictly necessary? I don't see any `tabindex` attribute in the template.

Reply:
You are right @dmcnamara-eng, I think these can be removed ✅